src/neo/Cryptography/Base58.cs

```diff
@@ -25,16 +25,16 @@ public static byte[] Decode(string input)
             // Leading zero bytes get encoded as leading `1` characters
             int leadingZeroCount = input.TakeWhile(c => c == Alphabet[0]).Count();
             var leadingZeros = new byte[leadingZeroCount];
-            var bytesWithoutLeadingZeros = bi.ToByteArray()
-                .Reverse()// to big endian
-                .SkipWhile(b => b == 0);//strip sign byte
+            var bytesWithoutLeadingZeros = bi.ToByteArray(isUnsigned: true, isBigEndian: true);
```

Comment:
It could still have leading zero for 255 or 255**2... is needed to.strip these as well? This method is only caring for lenght 1.

Reply:
isUnsigned doesn't produce leading zero.